packages/protocol/src/schema.proto

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright 2019 Wireline, Inc.
+//
+
+syntax = "proto3";
+
+package dxos.protocol;
+
+message Request {
+  bytes id = 1;
+  bytes error = 2;
+  bytes data = 3;
+  bytes options = 4;
+}
+
+// TODO(burdon): Rename. Move from protocol package.
+message PeerChatMessage {
```

Comment:
Shouldn't be here. Same with Presence. Separate extensions.

Reply:
WIP